src/main/java/com/osiris/autoplug/client/tasks/updater/server/TaskServerUpdater.java

```diff
@@ -165,6 +168,9 @@ private void doMCServerUpdaterLogic() throws Exception {
         // The update process will create the output file if it doesn't exist.
         updateBuilder.outputFile(outputFile);
 
+        // Set the working directory for the updater
+        updateBuilder.workingDirectory(downloadsDir); // TODO: FIX THIS
```

Comment:
Yeah this should be the directory where the classes get downloaded and the jar created.
The downloads dir should be fine for this.